samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
```

Comment:
>UITextField* [](start = 22, length = 12)

Why did you not use the TestEnabled- text field? As we add new features to these test controls, all these tests benefit. Did you have to workaround the setting the text programmatically?